TypeCobol.Test/Parser/Programs/Cobol85/DataGroupEmptyPGM.txt

```diff
@@ -1,6 +1,8 @@
 --- Diagnostics ---
-Line 7[8,14] <27, Error, Syntax> - Syntax error : A group item cannot be empty. OffendingSymbol=[8,14:MyGroup]<UserDefinedWord>
+Line 7[8,14] <27, Error, Syntax> - Syntax error : Copy declaration cannot use level 5 because a level 5 already exists in copy. OffendingSymbol=[8,14:MyGroup]<UserDefinedWord>
 Line 12[8,14] <27, Error, Syntax> - Syntax error : A group item cannot be empty. OffendingSymbol=[8,14:MyGroup]<UserDefinedWord>
+Line 28[4,7] <27, Error, Syntax> - Syntax error : Copy declaration cannot use level 1 because a level 1 already exists in copy. OffendingSymbol=[4,7:STR5]<UserDefinedWord>
```

Comment:
The error message is not clear:
For me "Copy declaration" is the copy itself. I'd rather say:
Group item STR5 is redudant (or not necessary), a level 01 already exists in copy.

Reply:
I close my remark. Proposition of fm-117 is better ;)